forge-gui/res/cardsfolder/upcoming/mold_folk.txt

```diff
@@ -0,0 +1,10 @@
+Name:Mold Folk
+ManaCost:1 B
+Types:Creature Fungus Warrior
+PT:1/1
+K:Lifelink
+A:AB$ PutCounter | Cost$ 1 Sac<1/Creature.Other;Artifact/another creature or artifact> | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Mold Harvest — {1}, Sacrifice another creature or an artifact: Put a +1/+1 counter on CARDNAME.
```

Comment:
another creature or an artifact> in the Cost to be consistent please

Reply:
Also needs `PrecostDesc$  Mold Harvest —`

Look at stuff like Blink Dog

Reply:
What are the factors that make the Precost parameter necessary? It doesn't really look any different that any other card with a similar ability.

Reply:
If there is a description before the cost, it should have `PrecostDesc$` for card detail display purposes

> It doesn't really look any different that any other card with a similar ability.
By "It" do you mean Blink Dog? Sorry, I'm not sure what you mean here.